base/install_area_detector.sh

```diff
@@ -96,3 +98,17 @@ git apply --directory ADSupport ${EPICS_MODULES_PATH}/nanohttp_stream.patch
 
 make -j${JOBS}
 make clean
+
+cd ..
+
+git clone https://github.com/cnpem/ssc-pimega
+git -C ssc-pimega checkout $LIBSSCPIMEGA_VERSION
+make -C ssc-pimega/c install
```

Comment:
It is possible to download the tarball for a specific git hash, and IMO we should do that.

Reply:
I like the idea. Would you rather support hash in `download_github_module` or use a directly link for this case in particular?

Supporting only tags seems better to force us to use only released code. On the other hand, it makes it harder to handle such cases in which the project does not use tags as a way to release.

Reply:
> Supporting only tags seems better to force us to use only released code. On the other hand, it makes it harder to handle such cases in which the project does not use tags as a way to release.

That's a debate I have seen a lot of times, lol

Extending the function to support hashes would be best if we know we are going to depend on it. Who's to say we are never going to need bug fixes for which a patch isn't feasible? I think I'd go for it.